config/typescript/react-transition-group.d.ts

```diff
@@ -0,0 +1,21 @@
+declare module '@material-ui/react-transition-group' {
```

Comment:
Why did we need to declare this module?

Reply:
Every package we consume needs types, however, most don't include them in the package. Polaris does include types, which is why we don't have a `@types/shopify__polaris-react`. `react-transition-group` does have a types package, which we're still consuming and I'm using to create type definitions for `@material-ui/react-transition-group` which doesn't have a `@types` package